README.md

```diff
@@ -230,3 +230,15 @@ either the individual or corporate Contributor License Agreement (CLA).
 Follow either of the two links above to access the appropriate CLA and
 instructions for how to sign and return it. Once we receive it, we'll be able to
 review/accept your PR.
+
+## Glossary
+The maintainers of the repository will tend to use specific jargon to describe
+concepts related to Magic Modules; here's a quick reference of what some of
+those terms are.
+
+Term          | Definition
+--------------|--------------
+tool          | One of the OSS DevOps projects Magic Modules generates GCP support in
```

Comment:
We use the word `provider` more often. I think it's important to make sure that users recognize that provider == tool.

Reply:
Right, I'll add that as well. Thanks!